src/thor/bidirectional_astar.cc

```diff
@@ -113,7 +104,6 @@ bool BidirectionalAStar::ExpandForward(GraphReader& graphreader,
                                        const GraphId& node,
                                        BDEdgeLabel& pred,
                                        const uint32_t pred_idx,
-                                       const bool from_transition,
```

Comment:
we no longer do this recursively so there is no calls that are from transition nodes